website/docs/interactors/2-built-in-dom.md

```diff
@@ -49,6 +52,14 @@ export default test('BigTest Runner')
   .assertion(Page.has({ title: 'BigTest Example App'}));
 ```
 
+### HTML
+
+The `HTML` interactor is not meant to be used directly in your tests but for _composing_ other interactors. How this interactor is used will make much more sense once you read the `Write Your Own Matchers` page. But for now you can think of the `HTML` interactor as the basic building block for all other interactors.
+
+We took many of the common HTML properties and interactions, such as `className` and `click`, and added them into the `HTML` interactor. This provides the convenience of not having to respecify these properties over and over again for each of your interactors.
+
+Take a look at the [source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts) for the `HTML` interactor to see which filters and actions were added.
```

Comment:
```suggestion
Take a look at the [source code](https://github.com/thefrontside/bigtest/blob/v0/packages/interactor/src/definitions/html.ts) for the `HTML` interactor to see which filters and actions are available.
```

I think there's a chicken and egg problem here - what order should you teach concepts in? Ideally, this information would be part of API docs and it could be glossed over in the guides. Until then, I think what you have is ok. You could refer to them as "features" if you wanted to be more general.

Reply:
Yeah, I don't feel good about mentioning filters or actions here when it hasn't been formally introduced yet.

I was trying to avoid the word `available` just in case people jump to the conclusion that these actions and filters are the _only_ ones they can use.

If we keep this `HTML` section in, should I just take out this particular paragraph?

Reply:
Yeah I think that solves the problem. You could still link to the source code in the rest of the prose. And I see that you are considering taking out the HTML explanation altogether or moving it, since it's not something we want people to use in the same way as the other built-ins.